beacon_node/eth1/src/deposit_cache.rs

```diff
@@ -143,24 +142,25 @@ impl DepositCache {
     ///
     /// ## Errors
     ///
-    /// - If `deposit_count` is larger than `range.end`.
+    /// - If `deposit_count` is larger than `end`.
     /// - There are not sufficient deposits in the tree to generate the proof.
     pub fn get_deposits(
         &self,
-        range: Range<u64>,
+        start: u64,
```

Comment:
Hey, out of interest why did you change this? :)

Reply:
So I was getting some clippy warnings re: this line 
https://github.com/sigp/lighthouse/blob/4b4bc6247da72a85154b3f7984f7ae917153db73/beacon_node/eth1/src/deposit_cache.rs#L363
saying "hey why are you not using `0..=n` instead of having `0..n + 1`. Classic range clippy warning.
However once I changed it, well it would not be a valid argument parameter since `get_deposit` was expecting a `Range`, not an `InclusiveRange`. So I thought about using the trait `RangeBounds`, and I did, and when I started refactoring `get_deposits` I thought "wait a minute, there's only one purpose to the range provided: know the start and the end of it".

So I thought it would be both more ergonomical and efficient (not acually generating a range before passing it to get_deposits) to use `start` and `end` u64 to do just that! :)